app/core/widgets/timeline/TimelineWidget.js

```diff
@@ -114,7 +114,7 @@ function(app, Backbone, $, _, __t, Directus, Utils, moment) {
         datetime: date,
         subject: subject,
         message: message,
-        comment_metadata: this.model.collection.table.id + ":" + this.model.get('id')
+        comment_metadata: this.model.collection.table.id + ":" + this.model.get(this.model.idAttribute)
```

Comment:
Hey @nums it looks good, but I will use `this.model.id` instead.

If it's possible change it, otherwise I will merge it and change it myself, if that sounds okay.